src/components/dialog-delete.js

```diff
@@ -19,15 +43,15 @@ class DialogDelete extends React.Component {
                     {this.props.typeName} settings?
                 </DialogTitle>
                 <DialogActions style={{ borderTop: 'none' }}>
-                    <Button onClick={this.props.onHandleDelete} color="primary">
-                        Delete
+                    <Button onClick={this.props.onHandleDelete} primary>
```

Comment:
A delete button should be [desctructive](https://ui-core.dhis2.nu/demo/?path=/story/button-destructive--default) not [primary](https://ui-core.dhis2.nu/demo/?path=/story/button-primary--default) see [the design guidelines](https://github.com/dhis2/design-system/blob/master/atoms/button.md)